frontend/server/configs.ts

```diff
@@ -50,14 +50,20 @@ export function loadConfigs(argv: string[], env: ProcessEnv): UIConfigs {
   const { staticDir, port } = parseArgs(argv);
   /** All configurable environment variables can be found here. */
   const {
-    /** minio client use these to retrieve minio objects/artifacts */
+    /** Object storage client credentials (SeaweedFS/MinIO compatible) */
     MINIO_ACCESS_KEY = 'minio',
     MINIO_SECRET_KEY = 'minio123',
     MINIO_PORT = '9000',
     MINIO_HOST = 'minio-service',
     MINIO_NAMESPACE = 'kubeflow',
     MINIO_SSL = 'false',
-    /** minio client use these to retrieve s3 objects/artifacts */
+    /**
+     * Name of the Kubernetes secret in each namespace containing object storage credentials.
+     * Used for multi-user credential isolation (Issue #12373).
+     * The secret should have 'accesskey' and 'secretkey' data fields.
+     */
+    MINIO_NAMESPACE_SECRET_NAME = 'mlpipeline-minio-artifact',
```

Comment:
Should we refactor this to OBJECT_STORAGE_SECRET NAME, or would it need changes in too many files ?